systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -1066,6 +1084,301 @@ void testRestoreOffsetsInConsumerGroup(ExtensionContext extensionContext) {
         assertDoesNotThrow(() -> ClientUtils.waitForClientTimeout(sourceConsumerName, namespaceName, 1));
     }
 
+    /**
+     * Test mirroring messages by MirrorMaker 2.0 over tls transport using scram-sha-512 auth
+     * while user Scram passwords, CA cluster and clients certificates are changed.
+     */
+    @SuppressWarnings({"checkstyle:MethodLength"})
+    @ParallelNamespaceTest
+    void testMirrorMaker2TlsAndScramSha512AuthRollAfterSecretsUpdate(ExtensionContext extensionContext) {
+        final String namespaceName = StUtils.getNamespaceBasedOnRbac(NAMESPACE, extensionContext);
+        final String kafkaClientsName = mapWithKafkaClientNames.get(extensionContext.getDisplayName());
+        String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        String kafkaClusterSourceName = clusterName + "-source";
+        String kafkaClusterTargetName = clusterName + "-target";
+        String topicSourceName = MIRRORMAKER2_TOPIC_NAME + "-" + rng.nextInt(Integer.MAX_VALUE);
+        String topicTargetName = kafkaClusterSourceName + "." + topicSourceName;
+        String kafkaUserSource = clusterName + "-my-user-source";
+        String kafkaUserTarget = clusterName + "-my-user-target";
+        String kafkaUserTlsSource = clusterName + "-tls-user-source";
+        String kafkaUserTlsTarget = clusterName + "-tls-user-target";
+        String kafkaTlsListener = "mytls";
```

Comment:
```suggestion
        String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
        
        String kafkaClusterSourceName = clusterName + "-source";
        String kafkaClusterTargetName = clusterName + "-target";
        
        String topicSourceName = MIRRORMAKER2_TOPIC_NAME + "-" + rng.nextInt(Integer.MAX_VALUE);
        String topicTargetName = kafkaClusterSourceName + "." + topicSourceName;
        
        String kafkaUserSource = clusterName + "-my-user-source";
        String kafkaUserTarget = clusterName + "-my-user-target";
        
        String kafkaUserTlsSource = clusterName + "-tls-user-source";
        String kafkaUserTlsTarget = clusterName + "-tls-user-target";
        
        String kafkaTlsListener = "mytls";
```
also, I would use `TestStorage`